### DISCUSSION TITLE
Cuda OOM

### DISCUSSION THREAD

**Discussion by SHAI-shamshad** [h2oai/wave#2236] [General]
I have created an experiment with nvidia gpu  machine with 24 GB of memory and the experiment ran successfully. Now when I am trying to run that same experiment(and keeping every config same as previous experiment). I am getting OOM. I am using mistral-7b-v0.1 base model. Any solution for that?

**Comment by mturoci** [comment]
Hey @ShamshadAhmedShorthillsAI. Which project are you referring to?

**Reply by SHAI-shamshad** [reply]
@mturoci  H2o LLM studio

**Reply by mturoci** [reply]
I see. In that case you want to post in https://github.com/h2oai/h2o-llmstudio/discussions. cc @pascal-pfeiffer
